samcli/commands/build/command.py

```diff
@@ -124,6 +124,20 @@
     type=click.Path(),  # Must be a json file
     help="Path to environment variable json file (e.g., env_vars.json) to pass into build containers",
 )
+@click.option(
+    "--build-image",
+    "-bi",
+    default=None,
+    multiple=True,  # Can pass in multiple build images
+    required=False,
+    help="Docker image URIs for building functions."
```

Comment:
I wouldn't overfit to Docker unless it's a Docker-specific technology. These are OCI containers and any runtime can run them. "Container image" should be enough.

Reply:
Updated.